modular_skyrat/modules/stone/code/stone.dm

```diff
@@ -18,6 +18,7 @@
 //	cost = 500
```

Comment:
Why is there so many lines like these that are commented out in this class declaration?

Reply:
genuinely I could not tell you why these are here or why they are commented out